crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/staticdata/enums/AppealContributionAmount.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.laa.crime.contribution.staticdata.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import uk.gov.justice.laa.crime.enums.AppealType;
+import uk.gov.justice.laa.crime.enums.AssessmentResult;
+import uk.gov.justice.laa.crime.enums.CrownCourtAppealOutcome;
+
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+public enum AppealContributionAmount {
+
+    NO_CONTRIBUTION(new BigDecimal(0)),
+    PART_CONTRIBUTION(new BigDecimal(250)),
+    FULL_CONTRIBUTION(new BigDecimal(500));
+
+    private final BigDecimal contributionAmount;
+
+    public static AppealContributionAmount calculate(AppealType appealType, CrownCourtAppealOutcome appealOutcome, AssessmentResult assessmentResult) {
+        if (assessmentResult.equals(AssessmentResult.PASS)) {
```

Comment:
I would consider swapping the order of the arguments in the `.equals()` to make this null safe
e.g.
instead of
`assessmentResult.equals(AssessmentResult.PASS)`
make it
`AssessmentResult.PASS.equals(assessmentResult)`

such that a null pointer exception will not be thrown is  `assessmentResult` is null.
Same feedback for the other enum based `.equals()` also.

Reply:
Thanks @stone-dj, will do!

Reply:
Done